site/content/gitops-quickstart/01-apply-gitops.md

```diff
@@ -0,0 +1,260 @@
+kube-system   kube-proxy-tbmdd          1/1     Running   0          3m
+$
+```
+
+## Applying GitOps
+
+The following command will set up your cluster with the `app-dev` profile,
+the first GitOps Quick Start. All of the config files you need for a
+production-ready cluster will be in the git repo you have provided and
+those components will be deployed to your cluster. When you make changes
+in the configuration they will be reflected on your cluster.
+
+> This is an experimental feature. To enable it, set the environment
+> variable `EKSCTL_EXPERIMENTAL=true`.
+>
+> Experimental features are not stable and their command name and flags
+> may change.
+
+```console
+EKSCTL_EXPERIMENTAL=true eksctl \
+        gitops apply app-dev-profile \
+        git-url=https://github.com/example/my-eks-config
+```
+
+- xxx: What will be installed on the cluster as part of this command
+
+This will set up Flux on your cluster and load GitOps Quick Start config
+files into your repo. It will use templating to add your cluster name and
+region to the configuration so that key cluster components that need those
+values can work (e.g. `alb-ingress`).
+
+## Your GitOps cluster
+
+- xxx: Describe what happens there
+- xxx: How users should check what is running at the end of the process
+  (e.g. get pods, port forward to kube dashboard, grafana, how they get
+  to the demo app that has been deployed)
+
+## Advanced setups
+
+`eksctl gitops apply` can largely be decomposed into
+
+1. `eksctl install flux`
+1. `eksctl generate config`
+
+So for more complex use cases, you will want to run these steps
+separately and on your own and adjust flags and options as necessary. The
+first command installs Flux and links it to a git repo that you provide.
+The second generates the config files from the GitOps Quick Start profile
+locally, so that you can edit them before pushing to your git repo.
+
+### Configuring Flux
+
+Here we will install [Flux](https://fluxcd.io), the Kubernetes GitOps
+operator in your cluster. It will take care of deployments for you.
+
+```console
+EKSCTL_EXPERIMENTAL=true eksctl install flux \
+    --git-url <git-url> \
+    --git-email <email-of-committer> \
+    --name <cluster-name>
+```
+
+Additional options to the command are explained in our docs on
+[`install flux`](/usage/experimental/gitops-flux/).
+
+After about a minute your cluster will have `flux` running, which will
+monitor your git repository once you added a deploy key to e.g. Github.
+The Flux Helm operator and Tiller will be deployed in your cluster
+too.
+
+This key can be found at the end of the output of the command, this
+might for example be:
+
+```console
+$ EKSCTL_EXPERIMENTAL=true eksctl install flux \
+    --git-url git@github.com:happy-gopher/flux-get-started \
+    --git-email happy@gopher.org \
+    --name wonderful-wardrobe-1565767990
+[...]
+[ℹ]  Flux will only operate properly once it has write-access to the Git repository
```

Comment:
This output should be in a separate code block - it's getting copied together with the command when I press the copy button.